dotcom-rendering/src/web/components/Callout/FileUpload.tsx

```diff
@@ -35,27 +83,54 @@ export const FileUpload = ({ formField, formData, setFormData }: Props) => {
 					...formData,
 					[formField.id]: stringifiedFile,
 				});
+				setChosenFile(event.target.files[0].name);
 			} catch (e) {
 				setError(
-					'Sorry there was a problem with the file you uploaded above. Check the size and type. We only accept images, pdfs and .doc or .docx files',
+					'Sorry, there was a problem with the file you uploaded above. Check the size and type. We only accept images, pdfs and .doc or .docx files',
 				);
 			}
 		}
 	};
-
 	return (
 		<>
-			<FieldLabel formField={formField} />
-			<input
-				data-testid={`form-field-${formField.id}`}
-				css={fileUploadInputStyles}
-				type="file"
-				accept="image/*, .pdf"
-				required={formField.required}
-				onChange={onSelectFile}
-			/>
-			<p>We accept images and pdfs. Maximum total file size: 6MB</p>
-			{!!error && <div css={errorMessagesStyles}>{error}</div>}
+			<div css={uploadStyles}>
+				<FieldLabel formField={formField} />
+				<div>
+					<label css={customUpload(format)} htmlFor={formField.id}>
```

Comment:
Could we use the FieldLabel here rather than having 2 labels for one component?

Reply:
We still do use the FieldLabel on line 97, it's just that for accessibility/CSS reasons we had to wrap the 'Choose File' button in a `<label>` tag so that the whole form field (including the label) was not clickable. This also enables us to neatly mark what field this button is related to via the `htmlFor` attribute.

Reply:
Is the field label a html label element? If we could wrap the Choose file button in the `FieldLabel` rather than the `label` the whole thing would be clickable and there wouldn't be two labels for one button which probably doesn't make sense sensibly

Reply:
Yep I get what you mean here. The problem I found was that when the whole thing was clickable you had unexpected behaviour when the 'Remove File' button appeared after uploading a file - it would mean that when you click that button you're immediately prompted to upload a file again. Just a weird UX. However, if you think there is a way around this I'd be more than happy to pair on it!

Reply:
Thanks @sophie-macmillan - I've removed this component now and have pushed the changes.